build.xml

```diff
@@ -159,6 +159,16 @@
     </target>
 
     <!-- Public API -->
+    <target name="spring4-soffit" description="Rebuild uPortal-soffit packages with Spring 4">
+        <property name="spring3version" value="3.2.9.RELEASE" />
+        <property name="spring4version" value="4.2.6.RELEASE" />
+
+        <replace file="gradle.properties" token="${spring3version}" value="${spring4version}"/>
```

Comment:
Using a hardcoded replace could cause issues if the `springVersion` is manually changed.
It could be safer to use the ant [propertyTask](https://ant.apache.org/manual/Tasks/propertyfile.html) or call the [propertyTask from gradle](http://stackoverflow.com/a/20549712).